kDriveCore/Data/Api/Endpoint.swift

```diff
@@ -133,10 +133,34 @@ public struct ProxyFile: AbstractFile, Sendable {
         self.id = id
     }
 
-    func resolve(using realm: Realm) throws -> File {
+    /// Resolve an abstract file within a `DriveFileManager`.
+    func resolve(within driveFileManager: DriveFileManager) throws -> File {
+        let file = driveFileManager.fetchObject(ofType: File.self) { realm in
+            fetch(using: realm)
+        }
+
+        guard let file, !file.isInvalidated else {
```

Comment:
More invalidated check here

Reply:
Yeah but this one is live so I keep it right ?